lms/static/sass/views/_bookmarks.scss

```diff
@@ -0,0 +1,144 @@
+// Rules for placing bookmarks and search button side by side
+.wrapper-course-modes {
+  border-bottom: 1px solid $gray-l3;
+  display: table;
+  width: 98%;
+  padding: 6px 3px;
+
+  > div {
+    display: table-cell;
+  }
+}
+
+
+// Rules for Bookmarks Button
+.courseware-bookmarks-button {
+  width: 40%;
+  padding: 0 3px;
+  vertical-align: top;
+
+  button {
+    &.is-active {
+      &:before {
+        content: "\f02e";
+        font-family: FontAwesome;
+      }
+
+      background-color: $white;
+    }
+
+    &.is-inactive {
+      &:before {
+          content: "\f097";
+          font-family: FontAwesome;
+      }
+
+      background: none;
+    }
+  }
+
+  .bookmarks-button {
+    background: none;
+    border-radius: ($baseline/4);
+    padding: 9px 8px 10px;
+    width: 100%;
+
+    &:hover {
+      background: none;
+    }
+
+    &:focus, &:active {
+      box-shadow: none;
+    }
+  }
+}
+
+
+// Rules for Bookmarks Results Header
+.bookmarks-results-header {
+  margin: 0;
+}
+
+// Rules for Bookmarks Results
+.bookmarks-results-list {
+  padding-top: $baseline;
+
+  .bookmarks-results-list-item {
+    display: block;
+    border: 1px solid $gray;
+    @include padding(($baseline/4), $baseline, ($baseline/2), $baseline);
```

Comment:
NIT: includes / extends should be ordered above the other  rules. +1 for using the RTL friendly styling here!